### DISCUSSION TITLE
添加下载 皮肤 和 资源包 和 数据包 和 光影 的功能

### DISCUSSION THREAD

**Discussion by LGCR837** [Meloong-Git/PCL#4740] [讨论]
在下载页面添加下载 皮肤 和 资源包 和 数据包 和 光影 的功能
皮肤源: 如LittleSkin(https://littleskin.cn)
资源包源: CurseForge、Modrinth、MC百科
数据包源: CurseForge、Modrinth、MC百科
光影源头: OptFine官网
顺便一提，把下载的搜索加上吧。
如图：《 PCL III 》.png
![PCL III](https://github.com/user-attachments/assets/3d1b6cba-c660-4d15-a562-a5deeea5a8d4)

**Comment by tangge233** [comment]
#4020 
#4360 
#4359 

请等待功能合并……

**Comment by allMagicNB** [comment]
#4020 #4359 已合入 PR Collection，可使用 https://github.com/allMagicNB/PCL2/actions/runs/10800930591/artifacts/1916599862 抢先体验。
（注意：此版本的更新系统尚未完善，请勿使用程序内的 `检查更新` 按钮，同时需遵守开源版本使用条例）

**Comment by Chiloven945** [comment]
MC 百科没有提供 API 下载，因此不能被算作下载源……
皮肤没记错已被拒绝……
